geotrek/api/v2/serializers.py

```diff
@@ -333,6 +334,41 @@ class Meta:
         fields = ('id', 'advice', 'filter', 'name', 'pictogram')
 
 
+class HDViewPointSerializer(serializers.ModelSerializer):
+    geometry = geo_serializers.GeometryField(read_only=True, source="geom_transformed", precision=7)
+    picture_tiles_url = serializers.SerializerMethodField()
+    trek = serializers.SerializerMethodField()
+    site = serializers.SerializerMethodField()
+    license = serializers.SlugRelatedField(
+        read_only=True,
+        slug_field='label'
+    )
+    create_datetime = serializers.DateTimeField(source='date_insert')
+    update_datetime = serializers.DateTimeField(source='date_update')
+
+    def get_picture_tiles_url(self, obj):
+        return build_url(self, obj.get_generic_picture_tile_url())
+
+    def get_trek(self, obj):
+        if obj.content_type == ContentType.objects.get_for_model(trekking_models.Trek):
```

Comment:
```suggestion
        if isinstance(obj, trekking_models.Trek):
```
prevent useless database query